swagger/swaggerDoc.yaml

```diff
@@ -0,0 +1,849 @@
+      responses:
+        200:
+          description: "Status 200"
+    post:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"color\": \"blue\",\n  \"name\": \"Wigglytuff Dratini\
+            \ Kakuna\",\n  \"regimen_items\": [\n    {\n      \"sequence_id\": \"\
+            7\",\n      \"time_offset\": \"123\"\n    }\n  ]\n}"
+      responses:
+        200:
+          description: "Status 200"
+  /api/regimens/{IntID}:
+    put:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"color\": \"blue\",\n  \"name\": \"something new\"\
+            ,\n  \"regimen_items\": [\n    {\n      \"sequence_id\": \"29\",\n   \
+            \   \"time_offset\": \"1555500000\"\n    },\n    {\n      \"sequence_id\"\
+            : \"29\",\n      \"time_offset\": \"864300000\"\n    },\n    {\n     \
+            \ \"sequence_id\": \"29\",\n      \"time_offset\": \"950700000\"\n   \
+            \ }\n  ]\n}"
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      produces:
+      - "text/plain"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+            description: "Empty Response\n"
+    parameters:
+    - name: "IntID"
+      in: "path"
+      required: true
+      type: "string"
+  /api/points:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    post:
+      consumes: []
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+          example: "{\n  \"pointer_type\": \"ToolSlot\",\n  \"name\": \"foo\",\n \
+            \ \"x\": 0,\n  \"y\": 0,\n  \"z\": 0,\n  \"pullout_direction\": 1\n}"
+      responses:
+        400:
+          description: "Status 400"
+          schema:
+            type: "object"
+            example: "{\n  \"error\": \"This is a JSON API. Please use _valid_ JSON.\"\
+              \n}"
+  /api/points/{IntID}:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    put:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"x\": 99,\n  \"y\": 87,\n  \"z\": 33,\n  \"radius\"\
+            : 55,\n  \"meta\": {\n    \"foo\": \"BAR\"\n  }\n}"
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      description: "IMHO: Bad Design - IntID is sth like an array of IDs?"
```

Comment:
I don't like this endpoint either but we need to account for devices that might have a few thousand points in their map and wish to delete a large number at once.

We originally only allowed one ID per request but hit performance issues on the client side because off all the HTTP request overhead. Can you propose an alternative solution for this?